rs/boundary_node/rate_limits/canister_client/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+use anyhow::{bail, Context, Error, Result};
+use candid::{Decode, Encode, Principal};
+use ic_agent::Agent;
+use rate_limits_api::{
+    v1::{RateLimitRule, SCHEMA_VERSION},
```

Comment:
This means that if the rules change (i.e. schema version changes), then this `v{i}` should be updated in the lib. I think the same should [be done in](https://github.com/dfinity/ic/blob/7136d8b228d6fbfbaf6ba9af0edc5f49e39a57e3/rs/boundary_node/ic_boundary/src/rate_limiting/fetcher.rs#L11) the `ic-boundary`, wdyt @blind-oracle ?